src/__tests__/providers-registry.contract.spec.ts

```diff
@@ -164,44 +281,20 @@ describe("Provider Registry Contract", () => {
               },
             }
           }
-        }, {
-          height: 1000
         }))
         .rejects
         .toThrowError("Initial stake must be zero.")
     });
-
-    it("throws if trying to add manifest with provider", async () => {
-      await expect(testEnv.interact<ProvidersRegistryInput>(caller, contractId,
-        {
-          function: "register-provider",
-          data: {
-            "provider": {
-              "adminsPool": [],
-              "profile": {
-                "name": "test-provider-1",
-                "description": "desc-1",
-                "url": "https://test-provider-1.ok"
-              },
-              "manifests": []
-            }
-          }
-        }, {
-          height: 1000
-        }))
-        .rejects
-        .toThrowError("Manifest should be added add with separate add-provider-manifest function.")
-    });
   });
 
-  describe("remove-provider function", () => {
+  describe("removeProvider function", () => {
     // adding two providers before execution of each test
     beforeEach(async () => {
       await testEnv.interact<ProvidersRegistryInput>(
         "bYz5YKzHH97983nS8UWtqjrlhBHekyy-kvHt_111111",
```

Comment:
Extracting this addresses as an "user1, user2, admin" constants in one place could improve readability

Reply:
yup